static/css/eval_retrieval_generation.scss

```diff
@@ -188,13 +189,12 @@ label {
 
   #form-container {
     border-top: solid 1px var(--gm-3-sys-light-outline-variant);
-    gap: 16px;
     padding: 16px 24px;
     flex-grow: 1;
     overflow: auto;
 
     .question-section .title {
-      font-size: 11px;
+      font-size: 16px;
```

Comment:
why change the question title font size? these were specs from adriana's mocks

Reply:
reverted